eureka/S3_data_reduction/niriss.py

```diff
@@ -74,14 +85,12 @@ def read(filename, f277_filename, data, meta):
     meta.time_units = 'BJD_TDB'
 
     # loads all the data into the data object
-    data.data = hdu['SCI',1].data + 0.0
+    data.data = hdu['SCI',1].data * hdu[0].header['EFFINTTM']
```

Comment:
I don't think we want to be multiplying by the integration time just yet.  This step happens in `b2f.rate2count()`.

Reply:
removed